src/net/sf/memoranda/EventsManager.java

```diff
@@ -161,6 +161,28 @@ public static Collection getEventsForMonth(CalendarDate date) {
 		return monthEvents;
 	}
 	
+	/**
+	 * Method: getEventsForCustomRange()
+	 * Inputs: CalendarDate date,
+	 * 		   int dateRange
+	 * Returns: Vector monthEvents
+	 * 
+	 * Description: Takes the current selected calendar day and makes a collection of all events
+	 * of that day and following custom day range. US-101.
+	 */
+	public static Collection getEventsForCustomRange(CalendarDate date, int dayRange) {  	
+    	int nextDay = 0;
+    	Vector customRangeEvents = new Vector();
+    	
+    	for (int j = 0; j < dayRange; j++) { 			
+    		Vector events = (Vector) EventsManager.getEventsForDate(date);
+    		nextDay++;
```

Comment:
Yeah, I was going to do that because nextDay was used in some other code that I deleted but I forgot.